File: ClipboardZanager/Tests/ClipboardZanager.Tests/Mocks/ServiceMock.cs
using ClipboardZanager.Shared.Services;

namespace ClipboardZanager.Tests.Mocks
{
    class ServiceMock : IService
    {
        public bool Reseted = true;

        public void Initialize(IServiceSettingProvider settingProvider)
        {
            Reseted = false;
        }

        public void Reset()
        {
            Reseted = true;
        }
    }
}

File: Android/app/src/androidTest/java/com/etiennebaudoux/clipboardzanager/mocks/ServiceMock.java
package com.etiennebaudoux.clipboardzanager.mocks;

import com.etiennebaudoux.clipboardzanager.componentmodel.services.Service;
import com.etiennebaudoux.clipboardzanager.componentmodel.services.ServiceSettingProvider;

public class ServiceMock implements Service {
    public boolean Reseted = true;

    @Override
    public void initialize(ServiceSettingProvider settingProvider) {
        Reseted = false;
    }

    @Override
    public void reset() {
        Reseted = true;
    }
}
